### ISSUE TITLE
error! helper: symbol to JSON conversion

### ISSUE THREAD

**Issue by loto** [ruby-grape/grape#1037]
Hi,

In 0.11.0:

``` ruby
error! :unauthorized, 401
```

Was returned as:

``` json
{ "error": "unauthorized" }
```

In 0.12.0 I get:

``` json
""unauthorized""
```

Does anyone else experience this?

**Comment by dblock** [comment]
Something is going on with either the error format or with what is being returned in a handled error. Read through https://github.com/intridea/grape/blob/master/UPGRADING.md carefully in case there's a clear answer (there're some things around errors), and if that doesn't help put up a project where this is reproduced?

**Comment by L-Oto** [comment]
I found nothing relevant in the upgrading documentation. I'll try to reproduce it in a small project.

**Comment by dm1try** [comment]
@dblock see https://github.com/ruby-grape/grape/blob/master/lib/grape/error_formatter/json.rb#L8 
```ruby
result = message.is_a?(String) ? { error: message } : message
```

**Comment by dblock** [comment]
I believe this is resolved via #1216 and some left-overs in #1309. Closing. Please reopen or add comments to #1309.
